src/commands/save-cache.yml

```diff
@@ -4,9 +4,13 @@ parameters:
     description: "The cache key to use. The key is immutable."
     type: string
     default: "go-mod"
+  path:
+    description: "Path to cache."
+    type: string
+    # /home/circleci/go is the GOPATH in the cimg/go Docker image
+    default: "/home/circleci/go/pkg/mod"
```

Comment:
`/home/circleci/` would only apply for CircleCI convenience images. What if we changed this to `~/` ?
I see it was originally set this way, wondering if anyone has complained yet.

Reply:
I don't think there should be any impact from that - interestingly, the `install` command doesn't set a GOPATH by default, although I haven't heard of any impact due to this.

Reply:
@gmemstr Unless I am misunderstanding, the issue would be on any non-circleci image this path will not exist. The orb should not be dependant on assuming it will be in the cimg/go Docker image